recipes/freealut/all/conanfile.py

```diff
@@ -0,0 +1,58 @@
+from conan import ConanFile
+from conan.tools.cmake import CMake, CMakeToolchain, cmake_layout
+from conan.tools.files import apply_conandata_patches, collect_libs, export_conandata_patches, copy, get, rmdir, rm
+import os
+
+required_conan_version = ">=1.54.0"
+
+class FreeAlutConan(ConanFile):
+    name = "freealut"
+    description = "freealut is a free implementation of OpenAL's ALUT standard."
+    topics = ("freealut", "openal", "audio", "api")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://directory.fsf.org/wiki/Freealut"
+    license = "LGPL-2.0-or-later"
+
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True]
+    }
+    default_options = {
+        "shared": True
+    }
+
+    def export_sources(self):
+        export_conandata_patches(self)
+
+    def configure(self):
+        self.options["openal-soft"].shared = self.options.shared
+
+    def layout(self):
+        cmake_layout(self, src_folder="src")
+
+    def requirements(self):
+        self.requires("openal-soft/[>=1.1]", visible=True, transitive_headers=True, transitive_libs=True)
```

Comment:
CCI currently allows version ranges to be used for only a small subset of packages. You will probably have to replace this with a specific version and a comment instead.
`visible=True` is the default and can be dropped.

Reply:
Done